google-cloud-vision/lib/google/cloud/vision/project.rb

```diff
@@ -288,19 +287,20 @@ def annotate *images, faces: false, landmarks: false, logos: false,
         # @param requests
         #   [Array<Google::Cloud::Vision::V1::AsyncAnnotateFileRequest | Hash>]
         #   Individual async file annotation requests for this batch.
-        #   A hash of the same form as
-        # `Google::Cloud::Vision::V1::AsyncAnnotateFileRequest` can also be
+        #   A hash of the same form as `
+        #     Google::Cloud::Vision::V1::AsyncAnnotateFileRequest` can also be
```

Comment:
What happened here? Might be a problem if there's a newline inside the pre block.

Reply:
That's a mistake. Fixing.